core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -910,6 +910,38 @@ secret_configs:
     - non_app_ci
     name: aws-chaos
     namespace: ci
+- from:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_app.ci_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_field: auth
+        email_field: email
+        item: cloud.openshift.com-pull-secret
+        registry_url: cloud.openshift.com
+      - auth_field: auth
+        email_field: email
+        item: quay.io-pull-secret
+        registry_url: quay.io
+      - auth_field: auth
+        item: quay.io/multi-arch
+        registry_url: quay.io/multi-arch
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+    .awscred:
+      field: .awscred
+      item: .awscred
```

Comment:
hello @hongkailiu @deepsm007 , I don't understand what value should be but in this `item` field, could you help me?
the vault I created for this cluster profile looks like this:
![image](https://github.com/openshift/release/assets/7559202/68eab968-bcab-4ccb-9d82-33f6d66e0f38)